modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,49 +5,63 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== SSO levels
+
+Organization admins can configure organization level SSO (org SSO) and tenant level SSO (tenant SSO). 
+SSO is a log-in method and access, roles, and permissions are dictated by role-based access control (RBAC).
+
+* *Org SSO:* Allows org admins to restrict how users log in when they are trying to access the org. 
+Access beyond login is managed via RBAC.
 
-[NOTE]
-====
-Accessing Aura with SSO requires:
- 
-* Authorization Code Flow with PKCE.
-* A publicly accessible Identity Provider (IdP) server.
-====
 
-== Console SSO
+* *Tenant level SSO:*  Impacts new database instances created within that tenant. 
+It ensures users logging in with SSO have access to the database instances within the tenant. 
+It depends on RBAC if the user can access and view or modify data within the database instances themselves. 
+For this level, the role mapping may be used to grant users different levels of access based on a role in their IdP (it *does not* give access to edit the tenant settings e.g. edit the tenant name / network access, or edit the instance settings e.g. rename an instance / pause and resume).
 
-Console SSO allows you to log in to the Aura Console using company IdP credentials and grants link:{neo4j-docs-base-uri}/cypher-manual/current/administration/access-control/built-in-roles#access-control-built-in-roles-public[Public Access privileges] to all instances in the tenant.
+== Login methods
 
-The following OpenID Connect (OIDC) certified Identity Providers (IdPs) are currently supported for Console-level Authentication:
+Login methods are different for each SSO level.
+Administrators can configure a combination of one or more of the login methods.
 
+*Organization level SSO supports:*
+
+* Email/password
+* Okta
 * Microsoft Entra ID
+* Google SSO
+
+An organization's administrator can add Aura as a login from a tile in an organization's Apps Dashboard.
+
+*Tenant level SSO supports:*
+
+* User/password
 * Okta
+* Microsoft Entra ID
 
-To enable Console SSO on your Aura Enterprise tenant(s), please https://support.neo4j.com/[raise a support ticket] including the following information:
+However, at the instance level you cannot disable user/password, but at the organization level you can disable email/password and Google SSO as long as you have at least one other custom SSO provider configured.
```

Comment:
Here's another instance of _instance level_.

Reply:
thanks, good spot